src/helpers/SwapHelper.ts

```diff
@@ -0,0 +1,10 @@
+import { MAX_SWAP_AMOUNT } from '@/constants/variables';
+
+export function getMaximumAllowed(userBalance: number, relayerBalance: number) {
+    let maxSwap = MAX_SWAP_AMOUNT;
+    
+    Math.min(userBalance, relayerBalance) > maxSwap ? null :
```

Comment:
I think we should refactor this into a more readable/user-friendly syntax, even though this is right it's hard for new readers to understand

Reply:
suggestion:

```
if ( Math.min(userBalance, relayerBalance) > maxSwap ) return MAX_SWAP_AMOUNT;

const maxSwap = userBalance > relayerBalance ? relayerBalance : userBalance;

return maxSwap
```

or something like that